cuebot/src/main/resources/conf/ddl/postgres/migrations/V32__Add_loki_job_fields.sql

```diff
@@ -0,0 +1,6 @@
+alter table job
+    add b_loki_enabled bool;
+
+alter table job
+    add str_loki_url varchar(256);
```

Comment:
I'm not completely sold on the idea of every job having these set. I agree runFrame is the best place to communicate this value from cuebot to rqd, but having to store this on the job table doesn't make too much sense to me.

`prepareRqdRunFrame` on dispatch support could read the logpath directly from jobLogUtil, keeping the database out of the loop